### DISCUSSION TITLE
Transfer music between players

### DISCUSSION THREAD

**Discussion by DesignsWithDavid** [music-assistant/support#1516] [Feature requests and ideas]
As an end user, I would like to be able to move the currently playing stream from one player to a next from within the UI.

This could be done by first syncing the players and then removing the initial one from the group.  

In the UI, it would seem logical to select the currently playing speaker, then tap and hold the name the speaker to be prompted to transfer playback.

**Comment by OzGav** [comment]
Please search before posting https://github.com/orgs/music-assistant/discussions/702
